meta.yaml

```diff
@@ -19,6 +18,7 @@ requirements:
   run:
     - python {{ python }}
     - pandas
+    - pytest
     - pyyaml
     - xlrd
```

Comment:
While we are at it -- do we still need the Excel reader?

Reply:
Yes. There is some test which needs excel files.

Reply:
It is because of `pensions.xlsx` use in `tax_transfer.py` and tests.

Reply:
Thanks. #73